src/utils/client.js

```diff
@@ -11,11 +11,13 @@ const client = new ApolloClient({
   fetch
 });
 
-const getItems = (query) =>
+const getItems = (query, variables = {}) =>
   client.query({
     query: gql`
       ${query}
-    `
+    `,
+    fetchPolicy: 'no-cache',
```

Comment:
I don`t think that all the queries need this. Consider use it only where you need it